src/app/app-common/PageSource/include/OpenKneeboard/PlainTextPageSource.h

```diff
@@ -65,19 +71,23 @@ class PlainTextPageSource final : public IPageSource {
   mutable std::vector<PageID> mPageIDs;
   std::vector<std::vector<winrt::hstring>> mCompletePages;
   std::vector<winrt::hstring> mCurrentPageLines;
-  std::vector<std::string> mMessages;
+  std::vector<std::string_view> mMessagesToLayout;
+  std::vector<std::string> mAllMessages;
 
   std::optional<PageIndex> FindPageIndex(PageID) const;
 
   float mPadding = -1.0f;
   float mRowHeight = -1.0f;
   int mColumns = -1;
   int mRows = -1;
+  uint32_t mFontSize;
```

Comment:
this should be a float to match the settings variable

Reply:
Done